src/main/java/org/tugraz/sysds/hops/rewrite/HopRewriteUtils.java

```diff
@@ -535,6 +537,32 @@ public static DataOp createTransientRead(String name, Hop h) {
 		return tread;
 	}
 	
+	public static DataOp createTransientRead(String name, MatrixBlock mb) {
+		DataOp tread = new DataOp(name, DataType.MATRIX, ValueType.FP64, DataOpTypes.TRANSIENTREAD,
+			null, mb.getNumRows(), mb.getNumColumns(), mb.getNonZeros(), UpdateType.COPY, 
+			ConfigurationManager.getBlocksize());
+		tread.setVisited();
+		tread.setBeginLine(1);
+		tread.setEndLine(mb.getNumRows());
+		tread.setBeginColumn(1);
+		tread.setEndColumn(mb.getNumColumns());
```

Comment:
Maybe we can introduce a method to set this info at once (e.g., all the line and column numbers)